packages/shorten-url/src/DidCommShortenUrlEvents.ts

```diff
@@ -0,0 +1,38 @@
+import type { BaseEvent } from '@credo-ts/core'
+
+export enum ShortenUrlEventTypes {
+  RequestShortenedUrlReceived = 'RequestShortenedUrlReceived',
+  ShortenedUrlReceived = 'ShortenedUrlReceived',
+  InvalidateShortenedUrlReceived = 'InvalidateShortenedUrlReceived',
+}
```

Comment:
```suggestion
export enum DidCommShortenUrlEventTypes {
  DidCommRequestShortenedUrlReceived = 'DidCommRequestShortenedUrlReceived',
  DidCommShortenedUrlReceived = 'DidCommShortenedUrlReceived',
  DidCommInvalidateShortenedUrlReceived = 'DidCommInvalidateShortenedUrlReceived',
}
```

Lately we are prefixing interfaces and events with "DIDComm"as much as we can to distinguish from other scopes like OpenID4VC.